Explain in README.md how to integrate sv_ttk with 3rd party modules (#143)
rdbende/Sun-Valley-ttk-theme: README.md

</details>

<details>
  <summary><b>Dark Mode title bar on Windows</b></summary>

  By default, sv_ttk doesn't change the title bar color on Windows when the theme is set to dark. But you can use [pywinstyles](https://github.com/Akascape/py-window-styles) to change the title bar color on Windows. Here's an example:
  
  ```python
  import pywinstyles, sys

  def apply_theme_to_titlebar(root):
      version = sys.getwindowsversion()

      if version.major == 10 and version.build >= 22000:
          # Set the title bar color to the background color on Windows 11 for better appearance
          if sv_ttk.get_theme() == "dark": pywinstyles.change_header_color(root, "#1c1c1c")
          else: pywinstyles.change_header_color(root, "#fafafa")
      elif version.major == 10:
          if sv_ttk.get_theme() == "dark": pywinstyles.apply_style(root, "dark")
          else: pywinstyles.apply_style(root, "normal")

          # A hacky way to update the title bar's color on Windows 10 (it doesn't update instantly like on Windows 11)
          root.wm_attributes("-alpha", 0.99)
          root.wm_attributes("-alpha", 1)

>     # Example usage (replace `root` with the reference to your main/Toplevel window)
>     apply_theme_to_titlebar(root)

Reviewer: ```suggestion
  # Example usage (replace `root` with the reference to your main/Toplevel window)
  apply_theme_to_titlebar(root)
```

nit

```

## Tips and tricks

> Sun Valley is meant to be a simple ttk theme wrapped in a PyPI package for convenience. I don't want to add other dependencies, nor features that would be trivial to implement if the user wants them, using 3rd party modules. But this theme can be easily integrated with them.

Reviewer: ```suggestion
Our intention is to keep the `sv-ttk` package as simple as possible, while making it easy to integrate with other libraries.
```

I don't like saying "I" or "the user" in places like this. Rather "we" and "you"

<details>
  <summary><b>Set the theme to the system theme</b></summary>
  
  You can use [darkdetect](https://github.com/albertosottile/darkdetect) for that. Here's an example:
  
  ```python

>   import tkinter, sv_ttk, darkdetect
>   from tkinter import ttk
> 
>   root = tkinter.Tk()
> 
>   button = ttk.Button(root, text="Click me!", style="Accent.TButton")
>   button.pack()
> 
>   sv_ttk.set_theme(darkdetect.theme())
>   root.mainloop()

Reviewer: ```suggestion
  import darkdetect

  sv_ttk.set_theme(darkdetect.theme())
```

I'd like to emphasize the simplicity of adding this functionality, and not repeat the example that is already presented above.

  ```

>   You just have to pass ```darkdetect.theme()``` to the ```sv_ttk.set_theme()``` function. It's that easy!

Reviewer: ```suggestion
  It's only a matter of an extra import and passing the result of `darkdetect.theme()` to ```sv_ttk.set_theme()`. It's that easy!
```

Same here

</details>

<details>
  <summary><b>Dark Mode title bar on Windows</b></summary>

  By default, sv_ttk doesn't change the title bar color on Windows when the theme is set to dark. But you can use [pywinstyles](https://github.com/Akascape/py-window-styles) to change the title bar color on Windows. Here's an example:
  
  ```python
  import tkinter, sv_ttk
  from tkinter import ttk

  # Function to set the title bar color
  def set_title_bar_color(root: tkinter.Tk | tkinter.Toplevel):
      theme = sv_ttk.get_theme()

      if get_windows_version() == 10:
          import pywinstyles

          if theme == "dark": pywinstyles.apply_style(root, "dark")
          else: pywinstyles.apply_style(root, "normal")

          # A hacky way to update the title bar's color on Windows 10 (it doesn't update instantly like on Windows 11)
          root.wm_attributes("-alpha", 0.99)
          root.wm_attributes("-alpha", 1)
      elif get_windows_version() == 11:
          import pywinstyles

          # Set the title bar color to the background color on Windows 11 for better appearance
          if theme == "dark": pywinstyles.change_header_color(root, "#1c1c1c")

>           elif theme == "light": pywinstyles.change_header_color(root, "#fafafa")

Reviewer: Oh, yeah. It can be a bit confusing how these random colors get here. I think I will define them in sv-ttk with constants, but until then they can stay like this.

  # Function to get Windows version			
  def get_windows_version() -> int:
      import sys

      if sys.platform == "win32":
         # Running on Windows
          version = sys.getwindowsversion()

          if version.major == 10 and version.build >= 22000: return 11 # Windows 11
          elif version.major == 10: return 10 # Windows 10
          else: return version.major # Other Windows version (like 7, 8, 8.1, etc...)
      else: return 0 # Not running on Windows
        
  root = tkinter.Tk()

  button = ttk.Button(root, text="Click me!", style="Accent.TButton")
  button.pack()

  sv_ttk.set_theme("dark")
  set_title_bar_color(root)
  root.mainloop()
```

  Note that on Windows 10, due to its limitations, you can only set the title bar's color to black for dark mode and white for light mode. On Windows 11 the title bar can be set to any color.

  Here's how the windows look after calling ```set_title_bar_color()```:

  <p align="center">
    <b>Windows 10</b>
    <br>
    <img src="assets/win10.png"/>
    <br><br>
    <b>Windows 11</b>
    <br>
    <img src="assets/win11.png"/>
  </p>
</details>

> 

Reviewer: I'm not sure about this code block. As I've already said I don't want to add extra complexity here, for example the type annotations.

I'd also highlight in the description that this is a Windows-only feature, and drop the `sys.platform == "win32"` check and move the `import pywinstyles` to the top.
I don't like how this requires different solutions on Windows 10 and 11, one of which is basically just a hack, but also don't want to hide the fact that it *can* work on Windows 10.

Can you simplify this part by removing non-relevant parts, as with the above code example, drop the platform check, and also move the version check in the same function that sets the header style.

I also think that `apply_theme_to_titlebar` would be a better name for this function.
